source/_posts/2016-10-14-backing-up-and-restoring-a-postresql-database.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Backing Up and Restoring a PostgreSQL Database"
+date: 2016-10-14 14:05:38 +0200
+comments: true
+author: Jovana Dačić
+categories: 
+  - Ruby and Rails
```

Comment:
@codalicious not sure if this is the best category. Misc should probably be the best fit, although we should think about some DevOps or similar category or something like that in near future.